code/modules/jobs/job_types/roboticist.dm

```diff
@@ -28,8 +28,12 @@
 
 	mail_goodies = list(
 		/obj/item/storage/box/flashes = 20,
-		/obj/item/stack/sheet/metal/twenty = 15,
-		/obj/item/modular_computer/tablet/preset/advanced = 5
+		/obj/item/stack/sheet/metal/fifty = 15,
+		/obj/item/ipcrevive = 10,
+		/obj/item/stack/sheet/plasteel/twenty = 5,
+		/obj/item/modular_computer/tablet/preset/advanced = 5,
+		/obj/item/stock_parts/cell/bluespace = 5,
+		/obj/item/borg/upgrade/syndicate = 1
```

Comment:
not entirely sure how i feel about this, maybe replace it with getting an overcharged yellow power cell?

Reply:
replaced with dilithium crystal